### ISSUE TITLE
BLE delivery time

### ISSUE THREAD

**Issue by hyeonuchoi** [huhu426/CS442_GM#4]
BLE module has been integrated in Navigation and Pedestrian application successfully.
And the BLE communication has been worked well, even the Navigation device is in Car. 
Test result is like below.
- Distance : 5 cm
- Obstacle : non
- Result : 
  1. Delivery time : 207ms, 215 ms, 209 ms, 211ms, 217ms => average 211.8 ms
  2. Amount of received packets : 8.5 pbs
- Distance : 50m
- Obstacle : Some trees
- Result : 
  1. Delivery time : 429 ms, 218 ms, 224 ms, 533 ms, 1282 ms => average 537.2 ms
  2. Amount of received packets : 42 packages / 6sec, 57/7.48, 44/8.5, 48/8.0, 37/4.36 => 6.6 pbs (packets per second)
- We found the BLE packets are blocked by vechicles (iron things) at today test.
  It may remain as a future work.
